### DISCUSSION TITLE
Is there no way to limit the text encoder training (steps/epochs) ?

### DISCUSSION THREAD

**Discussion by FGhrawi** [kohya-ss/sd-scripts#488] [Q&A]
Is there no way to limit the text encoder training (steps/epochs) ?
In most cases you do not need to train the text encoder as much as unet, and I do not see any option for doing that in this repo, or am I missing something ?

**Comment by kohya-ss** [comment]
The script for DreamBooth `train_db.py` has `--stop_text_encoder_training` option to stop Text Encoder training at a specified step.

Unfortunately other training scripts doen't have the option currently.
